extensions/wikia/AdEngine/js/slot/floatingMedrec.js

```diff
@@ -1,13 +1,17 @@
 /*global define*/
 define('ext.wikia.adEngine.slot.floatingMedrec', [
 	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.slot.service.viewabilityHandler',
 	'jquery',
+	'wikia.abTest',
 	'wikia.log',
 	'wikia.throttle',
 	'wikia.window'
 ], function (
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has 7 parameters which is greater than 4 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)